src/components/tour/tour_step.tsx

```diff
@@ -204,10 +205,25 @@ export const EuiTourStep: FunctionComponent<EuiTourStepProps> = ({
     size: 'xs',
   };
 
+  const isFooterActionArray = Array.isArray(footerAction);
+
+  const optionalFooterAction: JSX.Element = isFooterActionArray ? (
+    <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="flexEnd">
```

Comment:
i think you want `responsive={false}` and `wrap` for this. Currently this doesn't look great on mobile

<img width="498" alt="" src="https://user-images.githubusercontent.com/549407/203619144-94d0650f-0bf5-4220-810c-8079a75b0268.png">


```suggestion
    <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="flexEnd">
```

Reply:
Resolved in https://github.com/elastic/eui/pull/6384/commits/76dce60a73acd919373d3fcd89686fe3b9a1c319